src/University.rsc

```diff
@@ -722,7 +722,7 @@ Macro "University Directionality" (Args)
 
         mtx = null
         t_mtx = null
-        DeleteFile(pa_matrix_file)
+        //DeleteFile(pa_matrix_file)
```

Comment:
I'm assuming this was commented out for testing and left this way? Do you actually want to keep the pa matrix? If so, let's delete the line instead of commenting it.

Reply:
Yes, in one of the zone VMT calculations, we realized we need university PA matrix. I will go ahead and delete the comment line.